apps/civil/civil-sdt/prod-01.yaml

```diff
@@ -7,4 +7,5 @@ spec:
   values:
     java:
       environment:
+        PURGE_SCHEDULE_CRON: 0 15 0 * * 1,2
```

Comment:
Purge job needs to run 7 days a week. Suggest splitting it up as 1, 2 and 3 on one instance and 4, 5, 6 and 7 on the other.

Reply:
done